From: YOKOTA Hiroshi <yokota.hgml@gmail.com>
Date: Fri, 30 Jun 2023 17:07:53 +0900
Subject: Revert "PDF Output: Fix regression that caused large slowdowns when
 converting books with lots of internal HTML files to PDF"

Forwarded: not-needed

This reverts commit 6e990c3c1514357ec8a8cd6419c266cf682809bb.
---
 bypy/sources.json | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/bypy/sources.json b/bypy/sources.json
index 767bde8..ffc3bf9 100644
--- a/bypy/sources.json
+++ b/bypy/sources.json
@@ -415,9 +415,9 @@
     {
         "name": "podofo",
         "unix": {
-            "filename": "podofo-90ccd750ddf11be6853703ca432c0d96d89f19f2.tar.gz",
-            "hash": "sha256:9c7c9f18e70bfc3904c71048f12f48a047e718124cc626141867a2814e070d80",
-            "urls": ["https://github.com/podofo/podofo/archive/90ccd750ddf11be6853703ca432c0d96d89f19f2.tar.gz"]
+            "filename": "podofo-0.10.0.tar.gz",
+            "hash": "sha256:c9bf607fe4862b5cc6eac1754f0e39674c553350e8bbde68f5fff3e3eb2ed4bc",
+            "urls": ["https://github.com/podofo/podofo/archive/refs/tags/0.10.0.tar.gz"]
         }
     },
 
